dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventMapper.java

```diff
@@ -48,6 +51,42 @@
     })
 public interface EventMapper
     extends ViewMapper<Event, org.hisp.dhis.webapi.controller.tracker.view.Event> {
+
+  /**
+   * Events can be ordered by given fields which correspond to fields on {@link
+   * org.hisp.dhis.program.Event}.
+   */
+  static final Map<String, String> ORDERABLE_FIELDS =
+      Map.ofEntries(
+          entry("assignedUser", "assignedUser"),
+          entry("assignedUserDisplayName", "assignedUser.displayName"),
+          entry("attributeOptionCombo", "attributeOptionCombo.uid"),
+          entry("completedAt", "completedDate"),
+          entry("completedBy", "completedBy"),
+          entry("createdAt", "created"),
+          entry("createdBy", "createdBy"),
+          entry("deleted", "deleted"),
+          entry("enrolledAt", "enrollment.enrollmentDate"),
+          entry("enrollment", "enrollment.uid"),
+          entry("enrollmentStatus", "enrollment.status"),
+          entry("event", "uid"),
+          entry("followup", "enrollment.followup"),
+          entry("occurredAt", "executionDate"),
+          entry("orgUnit", "organisationUnit.uid"),
+          entry("orgUnitName", "organisationUnit.name"),
+          entry(
+              "program",
+              "enrollment.program.uid"), // TODO(ivo) this is what we do in the export. I assume we
```

Comment:
@enricocolasante do you have an opinion on whether we should use

`"enrollment.program.uid"` or `"program.uid"`

Reply:
I changed the todo to `TODO(TECH-1606)`.
I will keep this as it is for now and investigate this separately.

Maybe this info tells you whats going on:
* "enrollment.program.uid" turns to DB alias "p_uid"
* which refers to the uid in program table which we get via joins in  
https://github.com/dhis2/dhis2-core/blob/b157e811a066a41d716cfc7d3d6ae3ae6408e276/dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/JdbcEventStore.java#L784-L787